purchase_stock_picking_invoice_link/models/purchase_order.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models
+from odoo.tools import float_compare, float_is_zero
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "purchase.order.line"
```

Comment:
@pedrobaeza too much twin: should be class PurchaseOrderLine(moels.Model):

Reply:
Hehe, indeed. Although it doesn't affect on Odoo itself, let's name it correctly.

Reply:
@rvalyi changed